### ISSUE TITLE
Multiple Select and Action

### ISSUE THREAD

**Issue by Darknior** [gazpan/WebtroPie#17]
I have try these function : 

```
Ctrl + click to add to selection list
Shift + click to add range between last click and click to selection list
Shift + Ctrl + click to select all or none
```
And all is working fine, it's cool :)

But this one is not working for now :
`Right click for context menu which applies to either the selection list or single game if the mouseover game is not selected`

I think it is the only way to work on a multiple selection.
But to do what ? What can we do on a multi select files ?
If you have already think to it and some ideas can you share them here ?

**Comment by Darknior** [comment]
I think the only action we can try are : 

- Mass RENAME but how ? What is the interest ? Maybe with REGEX but it is complicated.
- Mass DELETE, it is an good idea, to remove XML entry and the ROM file.

**Comment by gazpan** [comment]
... yes, you are right, I started to develop that and was hoping to get some ideas from the community

some ideas :- 
a) Move Game ROMs to subdirectory ... and then select which one, (this was one of the things that I originally wanted WebtroPie to do but Favourites and Collections made this not necessary for me anymore)
b) Delete Games from Game list  ... and Y/N to delete ROMs
c) Add Games to Custom Collection ... and then select which collection (I think this in combination with filtering + selecting would be very helpful in setting up collections)
d) Scrape ROMS ... although not sure about adding scraping any more since sylphs scraper is now better integrated into ES
e) ... ?

Custom collections is on my to do list, I restructured the carousel code so that the carousel options could change depending on which custom collections that the theme supported as it either has a logo or is in the collections list. So, just a bit more work to do there, but not too bad.

**Comment by Darknior** [comment]
a) Yes now it is really less interesting.
b) Yes excellent for this.
c) Woaw, yes if you can work on **Collection** from your Webtropie, it can be better than do in on PI.
I never think to it, i love this idea.
d) **Scrapping**, yes i have think to it, i love scrapping, i use **XML Scrapper** every time to began a collection before finish it with your Webtropie.
It is some work i think, but if one day you can add the SS Scrapper it will help many people.
**Screen Scrapper** is the best one i think, and you scrap in _FR, US, DE, SP_ ... many guys from many countries working on it. It is better than the old RETROPIE scrapper :(
https://www.screenscraper.fr
And they have an API you can use :)

WOAW Great news if you will also add the **Custom Collection** <3

**Comment by Darknior** [comment]
For the SS API we must ask for a KEY here : 
http://www.screenzone.fr/forum/viewtopic.php?f=434&t=709

To make API Request, you must use maybe PHP to make GET request on URL like this : 
https://www.screenscraper.fr/api/jeuInfos.php?devid=xxx&devpassword=yyy&softname=zzz&output=xml&ssid=test&sspassword=test&crc=50ABC90A&systemeid=1&romtype=rom&romnom=Sonic%20The%20Hedgehog%202%20(World).zip&romtaille=749652

**Comment by gazpan** [comment]
I would love to have a go at that, I've been meaning to add scraping for a long time, it's actually one of the milestones I have to get WebtroPie to Version 1.0. !

.. I imaging, a bit like we've been working on with drag and drop images, we drag a Rom file on a game list and as well as uploading the rom to the rom directory it could automatically scrape too, so with one action most things are already done. I know most roms are done in sets anyway but this may help when making an individual list.

Wow, that scraper returns a lot of detailed information!, I could easily have an ajax call sent direct from the browser in js, I would have to read the API but I'm guessing the crc is required? so I would probably use php to calc that.

I had a look at the screen zone page, it says that if you are a developer and want to use their api to contact them and then points to a thread which is a dead link, ... mon français est terrible! .. what should I do ?

**Comment by Darknior** [comment]
Yes scrapper is important for the future, but for the moment i think if you finish to fix the small bug list and add the image PATH to Popup settings ... all will be ready to use for most of the users.

1. Pause, delay or control video preview : Easy
2. Game Editor: Show full size image and image dimensions : Not so long with a Tool-tip ?
3. Last played date should show as 'Time ago' format : Convert the date and think of Translation
4. Images upload not working : Add on menu the 3 files Path and Image Max Size.
5. Web access in settings

I think if you finish these 5 point, WOW you can release a v0.9 !
I write 0.9 because maybe you want work again on Skinning before 1.0 ?

You can make MOVE, DELETTE, Translation, etc ... for 1.1 1.2 .5 ...

And take the Scrapper for final 2.0 !

* Automatic Scrap when adding a ROM is really a cool idea :)

* Yes this Scrapper is really fantastic and the DEV is cool, he add many things i ask for, like Mass Import from ES XML, all the work i do with your Webtropie, i can import it to the SS DB ;)
Like i already upload many of may final image stuff made with Photoshop.
If you have questions, @MarbleMad will always answer you on https://kiwiirc.com/client/irc.freenode.net/#screenzone

* Yes you must ask for a KEY and they will give it to you, it is not a problem i think.
So bad we don't have a PI Scrapper to to work with them on RETROPIE, some one made one for RECALLBOX, but i'm not enough strong to convert it for RETROPIE.
Ask the KEY on the IRC, is is the best way i think :)
But MarbleMad is there on French day, not the night, after 19h he go away ...
